### PR TITLE
metadata updates for Kazakhstan and Russian Federation (prefix 7)

### PR DESCRIPTION
**Pull request by Logrus86**
english carrier data significantly updated for Kazakhstan and Russia (prefix 7)
russian carrier data added for Kazakhstan and Russia (prefix 7)
geocoding data significantly updated for Kazakhstan and Russia (prefix 7)
timezones data significantly updated for Kazakhstan and Russia (prefix 7)

### CODE REVIEW

File: resources/carrier/en/7.txt [hunk 1]
```diff
@@ -12,13 +12,582 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-7701|Kcell
-7702|Kcell
+# Sources:
+# Kazakhstan : https://ru.wikipedia.org/wiki/??????????_????_?????????_??????????
```
Comment (dlyongemallo) [inline resources/carrier/en/7.txt]: This was garbled.

Comment (Logrus86) [inline resources/carrier/en/7.txt]: I'm sorry for that! Thanks for the notice! Fixed.

**Comment by vladrusu** [comment]
Great! Please merge.

**Comment by keghani** [comment]
Thanks @Logrus86 !
I see you've already signed the CLA so we can start reviewing this PR. Sorry we missed it since April, thanks for the ping @vladrusu !

@Logrus86 Can you share all the sources in this thread? I saw some in the files but want to be sure I got them all:

```
Wikipedia
http://telecom.kz/page/single/kazahstan
http://www.rossvyaz.ru/opendata/7710549038-Rosnumbase/
```

* Wikipedia is often outdated on numbering plans so we can't accept that. Can you find an official doc or website that covers those prefixes?
* The Kazakhstan link is wrong, please give the correct link.
* Where in the Russia link can I download the doc/s you used?
* Any others I missed?

**Comment by keghani** [comment]
Hi!

We've updated the process for making contributions and are not accepting pull requests for metadata updates at this time.

Please help us fix this issue. See our [contribution guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md) and file an issue with the relevant details.

**Comment by Logrus86** [comment]
Hello @keghani !

- KZ link was changed while pull request was pending. There is the up to date one: http://telecom.kz/catalog/kazahstan.171777/5
- At Russia link you can download csv files with national numbering plans:
http://www.rossvyaz.ru/opendata/7710549038-Rosnumbase/Kody_ABC-3kh.csv
http://www.rossvyaz.ru/opendata/7710549038-Rosnumbase/Kody_ABC-4kh.csv
http://www.rossvyaz.ru/opendata/7710549038-Rosnumbase/Kody_ABC-8kh.csv
http://www.rossvyaz.ru/opendata/7710549038-Rosnumbase/Kody_ABC-9kh.csv
While I was working on my own project, I have create a parser and parse thoose 4 files.
That's it. (I know it's not required anymore but let it be there for now)

As I can see at [contribution guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md), it's not possible to update metadata through the pull requests anymore? I need to create a issue with all this data now? 
Will it be enough to create 1 issue?

Or 4 issues:
1) english carrier data significantly updated for Kazakhstan and Russia (prefix 7)
2) russian carrier data added for Kazakhstan and Russia (prefix 7)
3) geocoding data significantly updated for Kazakhstan and Russia (prefix 7)
4) timezones data significantly updated for Kazakhstan and Russia (prefix 7)

Or 8 issues: 4 for KZ and 4 for RU ?

**Comment by keghani** [comment]
Thanks for reading the guidelines @Logrus86 and the detailed explanation :)

In this case since KZ and RU share a country calling code (and therefore the carrier and geocoding files) you can go ahead and file a single issue. It's probably easiest to update them in one go.

**Comment by mknecht** [comment]
Was this addressed in a different issue, or is the Kazakhstan data still largely outdated?

**Comment by penmetsaa** [comment]
@mknecht
We are tracking this issue in #1002 and working on it.
We'll update in #1002 when it's released.

Closing this PR as we are not accepting PRs for metadata updates.

**Pull request closed without merging**